docs/DeveloperGuide.md

```diff
@@ -471,6 +473,59 @@ MSS:
 * 4d. The user closes QuickCache
 
   Use case ends.
+  
+**Use case: UC08 - Update flashcard save file**
+
+**Actor: QuickCache**
+
+MSS:
+
+1. QuickCache accepts update request
+2. QuickCache update save file
```

Comment:
updates* save file

Reply:
Noted, I have made the changes and standardised the format of our use cases